tests/test_request_task.py

```diff
@@ -0,0 +1,127 @@
+"""
+Helpful documentation on how to setup celery test fixtures: https://medium.com/@scythargon/how-to-use-celery-pytest-
+fixtures-for-celery-intergration-testing-6d61c91775d9.
```

Comment:
URL looks split on 2 lines with explicit newline

Reply:
In fact, the `make fix` did this... I needed to include an empty line to avoid this behavior